crates/sui-pg-temp-db/src/lib.rs

```diff
@@ -10,7 +10,7 @@ use std::{
     process::{Child, Command},
     time::{Duration, Instant},
 };
-use tracing::trace;
+use tracing::{event_enabled, info, trace};
```

Comment:
basically the tempdb.rs in sui-indexer and reorg the codes per Ashok's comment earlier.

Reply:
FWIW, if it's going in its own crate, the comment about ordering is less important. It was more relevant when it was sitting next to other files that have the same structure, for uniformity. (Also fine by me to make the change!)